pkg/asset/internal/templates.go

```diff
@@ -243,14 +247,15 @@ metadata:
   name: kube-proxy
   namespace: kube-system
   labels:
-    k8s_app: kube-proxy
-    version: v1.3.6_coreos.0
+    k8s-app: kube-proxy
+    update-controller-managed: "true"
+  annotations:
+    update-controller.alpha.coreos.com/priority: "100"
```

Comment:
Just for (easier to figure out) determinism, can we skew this from the kubelet (proxy first, then kubelet)